R/fabric.R

```diff
@@ -0,0 +1,99 @@
+#' @title Fabric
+#'
+#' @description Fabric provides a set of tools to allow you to mark up
+#'   your `xaringan` slides. Click the "edit" icon to begin. Use the
+#'   eraser to remove select markup, or the trash can to clear the entire
+#'   canvas. NOTE: You will not be able to navigate slides while in draw or
+#'   erase mode.
+#'
+#' @details You may toggle the visibility of the canvas toolbox by pressing
+#'   **D** at any time. Markup will persist when changing slides as long as the
+#'   canvas isn't cleared. You may save a permanent copy of the slides with the
+#'   markup by saving your presentation (e.g. using Chrome > File > Print).
+#'
+#' @param pen_color Pen color (default is "black"). Users may choose any
+#'   valid CSS Hex, RGB, or HSL color.
+#' @param pen_size Pen size (default is 3).
+#' @param eraser_color Eraser color (default is "rgba(0, 0, 0, 0.1)").
+#'   Users may choose any valid CSS Hex, RGB, or HSL color.
+#' @param eraser_size Eraser size (default is 10 times `pen_size`).
+#'
+#' @return An `htmltools::tagList()` with the fabric dependencies, or an
+#'   [htmltools::htmlDependency()].
+#' @section Usage: To add a fabric canvases to your `xaringan` presentation,
+#'   add the following code chunk to your slides' R Markdown file.
+#'
+#'   ````markdown
+#'   ```{r xaringan-fabric, echo=FALSE}
+#'   xaringanExtra::use_fabric()
+#'   ```
+#'   ````
+#'
+#' @name fabric
+NULL
+
+#' @describeIn fabric Adds a markup canvas to your xaringan slides.
+#' @export
+use_fabric <- function(
```

Comment:
I agree, happy to change things over to `scribble`.

Reply:
I was going to make a similar suggestion since end users don't need to know that fabric is being used in the implementation.  I like scribble.  (I was going to suggest draw, but I kind of like scribble.)